src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Day/Cells/DiaryDayEntryTableViewCell.swift

```diff
@@ -46,7 +46,12 @@ class DiaryDayEntryTableViewCell: UITableViewCell, UITextFieldDelegate {
 		label.text = cellModel.text
 		label.font = cellModel.font
 		
-		headerStackView.accessibilityLabel = cellModel.text
+		
+		if cellModel.parametersHidden {
+			headerStackView.accessibilityLabel = cellModel.text
+		} else {
+			headerStackView.accessibilityLabel = String(format: AppStrings.ContactDiary.Day.personSelected, cellModel.text)
+		}
```

Comment:
```suggestion
		headerStackView.accessibilityLabel = cellModel.text
		if cellModel.parametersHidden {
			headerStackView.accessibilityTraits = [.button]
		} else {
			headerStackView.accessibilityTraits = [.button, .selected]
		}
```

Have you tried setting the accessibility traits instead of extending the label manually?